### ISSUE TITLE
This simply does not work. 

### ISSUE THREAD

**Issue by CSTrombone** [mikelor/Chaaos#1]
I added the proper meta-data to AndroidManifest.xml and created "automotive_app_desc". Despite everything being done correctly, when I rebuild my solution, I get these errors: "resource xml/automotive_app_desc not found" and "failed processing manifest"

My project is EXACLTY the same as yours. Literally just copied everything verbatim. "xml/ automotive_app_desc not found" every single time.

**Comment by mikelor** [comment]
When you say copied verbatim, did you do a "git clone https://github.com/mikelor/Chaaos" and then compile?

Or did you start from scratch?

**Comment by CSTrombone** [comment]
> When you say copied verbatim, did you do a "git clone https://github.com/mikelor/Chaaos" and then compile?
> 
> Or did you start from scratch?

I started from scratch and it didn't work. After I cloned it from git I realized the problem, the method from the YouTube video does not work in .NET 8.0. 

It only seems to work in 7.0. 

I'd like to leave this comment here for anyone else having this problem, seeing as to how this is the only real introductory information for how to develop for Android Auto.
